server/engine/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java

```diff
@@ -82,6 +84,10 @@ public final class CobolVisitor extends CobolParserBaseVisitor<List<Node>> {
 
   private final TextExtractionState text;
 
+  private static final Pattern JAVA_DIRECTIVE_PATTERN =
+      Pattern.compile(
+          "(?i)\\s*(>>)(\\s*)(JAVA-CALLABLE|JAVA-SHAREABLE\\s+(?:ON|OFF))(?:\\s+(.+))?\\s*$");
```

Comment:
```suggestion
          "(?i)\\s*>>(\\s*)(JAVA-CALLABLE|JAVA-SHAREABLE\\s+(?:ON|OFF))\\s*(\\S.*)$");
```
The first group is not used.